### PR TITLE
docs: Add first reflexion about RIK CLI

### PR DESCRIPTION
**Pull request by jlabatut**
@sameo 
@NelopsisCode

### CODE REVIEW

File: docs/rep/cli/README.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+replicas: 3
+image: nginx:1.9.0 or url
+env:
+  - name: ENV
+    value: prod
+ - name: USER
+    value: john
+ ...
+```
+
```
Comment (thomasgouveia) [inline docs/rep/cli/README.md:71]: Please take in consideration that for the first releases there is no Docker. Only Bare metal apps. So as a member of the Riklet team (workload executor on nodes), we need to have a more complex workload description in order to run for example an nginx server. ( Repository ? Is a binary ? Is a package ? Start command ...)

File: docs/rep/cli/README.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+## Next RIK versions
+
+- Authentication system for multi tenant usage ? (`rik register`, `rik login`)
+- Possibility to use the CLI in an imperative way ?
+- Provide multiple nodes ? Node management (create, list, delete).
```
Comment (thomasgouveia) [inline docs/rep/cli/README.md:76]: This is not the CLI responsibility (For example we can't add a new node with a kubectl command)

File: docs/rep/cli/README.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  name: cluster name
+  server: API-address
+  auth-token: ApyTUlpd0JnNmRXVGpnaG5BNGhMRXVYVldNSHd1d3gxT0Nac3RmMWpqdEIv
+```
+
+The CLI will parse this file on every command, to get the API address and the auth token.
+Each request will contain the authentication token in the headers, and the YAML file in the payload.
+
+## Commands
+
```
Comment (sameo) [inline docs/rep/cli/README.md:27]: `pod` is a `riktl` command. You should list and document all supported commands and then for each command, list and document each subcommand. For example:

## Commands

`riktl` supports the following commands:

* `pod` for managing pods in a RIK cluster.

### `riktl pod`

The `pod` command allows for managing pods through a RIK cluster. It provides sub commands for listing, creating, and deleting pods:

#### `pod list`

bla bla bla

#### `pod create`

foo bar

#### `pod delete`

delete 'em all

File: docs/rep/cli/README.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+```
+cluster:
+  name: cluster name
+  server: API-address
+  auth-token: ApyTUlpd0JnNmRXVGpnaG5BNGhMRXVYVldNSHd1d3gxT0Nac3RmMWpqdEIv
+```
+
+The CLI will parse this file on every command, to get the API address and the auth token.
+Each request will contain the authentication token in the headers, and the YAML file in the payload.
```
Comment (sameo) [inline docs/rep/cli/README.md:24]: What's the relationship between a tenant and an auth token?

File: docs/rep/cli/README.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+  Help on a specific entity
+  `> rik pods|pod|po --help|-h`
+
+## Workload description
+
+> _workload.yaml_ :
+
+```
+kind: pod
```
Comment (sameo) [inline docs/rep/cli/README.md:60]: Are there other kind of workload than a pod? If now, why not calling it a workload then?

File: docs/rep/cli/README.md (hunk 2 above)
Comment (sameo) [reply docs/rep/cli/README.md:76]: I'd agree with that. This is the cluster's operator to add nodes to it.

Comment (NelopsisCode) [reply docs/rep/cli/README.md:76]: Ok! We remove it

File: docs/rep/cli/README.md [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+
+## Commands
+
+---
+
+### Create a workload
+
+- In a full command line
+
+  `riktl create workload --name NAME --image IMAGE ...`
```
Comment (sameo) [inline docs/rep/cli/README.md:57]: Do we really want to support that one?

**Review by sameo — COMMENTED** [review]
Except for one minor comment, this looks pretty good to me.

**Comment by sameo** [comment]
@thomasgouveia PTAL as well.

**Review by sameo — APPROVED** [review]

**Pull request merged**